### DISCUSSION TITLE
Replace 4.6.1 with 4.7 - settings are inherited in web part?

### DISCUSSION THREAD

**Discussion by rikehuman** [microsoft-search/pnp-modern-search#2378] [Q&A] [answered]
I have version 4.6.1 on tenant level and I want to upgrade to 4.7. What happens with all the settings I made in my web parts of 4.6.1? I use the web part with very special queries on 40 sites. I hope I will not have to "save" those settings and recreate them after deploying to 4.7.
Thanks for your help!

**Comment by wobba** [comment, marked as answer]
An upgrade should work fine by installing the new version over the old one. All settings are backwards compatible. I do suggest to perhaps wait for v4.8 before upgrading.

**Reply by rikehuman** [reply]
Great, thank you so much!
